docs/doc_entrypoint.md

```diff
@@ -1,10 +1,3 @@
-# Clients for osparc-simcore public web API
+<h1 align="center">Osparc-Simcore</h1>
 
-- API version: 0.4.0
-- Package version: 0.5.0
-
-## Clients
-
-Currently there is only a single client:
-
-- [Python](../clients/python/docs/README.md)
+[Osparc](https://osparc.io/) is a platform for developing, running and sharing life sience simulations. This site documents clients for interacting with osparc programatically. Choose your favorite programming language in the top-right navigation bar and get started.
```

Comment:
I would replace the link with:

[o²S²PARC](https://sparc.science/resources/4LkLiH5s4FV0LVJd3htsvH).

And re-use the text we have in the information page.
"is an open, cloud-based platform for the development, execution and sharing of computational models, simulations, and data analysis pipelines, and the presentation of results"

Reply:
Thanks for pointing this out. I have fixed this.